### PR TITLE
Add some libs & Minor fixes

### PR DESCRIPTION
**Pull request by kawas44**
Hi,

Could you please review my PR?
I am available to make any changes you find necessary.

The PR content:
  - Minor formatting
  - Fix Olical/conjure shadowing macourtney/Conjure
  - Charred lib is also for CSV files
  - Remove immutable-bitset replaced by data.int-map
  - Add thurber for Google Dataflow
  - Add tools.build
  - Add bifurcan for alternative data structures
  - Add exoscale/ex for exception handling
  - Add circleci/bond spying & stubbing lib
  - Add clj-ulid that implements ULID

Regards

### CODE REVIEW

File: projects.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (weavejester) [inline projects.yml:1]: Why this line?

Comment (kawas44) [reply projects.yml:1]: Just my yaml linter telling me to start a yaml document with it. But we can safely remove it, if you want?

Comment (weavejester) [reply projects.yml:1]: It's optional, so I think remove it from the commit to minimize changes.

Comment (kawas44) [reply projects.yml]: ok, I have force pushed the change.

Comment (kawas44) [reply projects.yml]: do you want me to change anything else?

**Comment by weavejester** [comment]
I think that's all good. Thanks!

**Pull request merged**